City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-4151: Remove taxonomies

Removed 'Emphasis' and 'Educational mission' taxonomies and entity reference fields from TPR Unit entities.

How to test:

1. Set up KASKO development environment
2. Checkout this branch: `git checkout UHF-4151_remove_taxonomies`
3. Import config: `make drush-cim`
4. Make sure that
  4.1 "Emphasis" and "Educational mission" taxonomies have been removed
  4.2 Entity reference fields referencing the mentioned taxonomies have been removed from TPR Unit entities
  4.3 Permissions regarding the mentioned taxonomies have been removed
